Assets/Scripts/System/GM/DataManager.cs

```diff
@@ -62,14 +63,13 @@ public void GetVideos()
                     introVideo = file.FullName;
                 else if (file.Name.ToLower().Contains("background"))
                     launcherBackground = file.FullName;
-                else if (file.Name.ToLower().Contains("attract"))
-                    attractFiles.Add(file.FullName);
+                else
+                    attractItems.Add(new AttractItem(file.FullName));
             }
 
             else
```

Comment:
This is just a style thing, but I feel like if an `if` block uses braces, then the `else` should, too. Even if it's just one line.